python/covid_web/covid_web/country_dashboard.py

```diff
@@ -110,7 +111,7 @@ def create_dashboard(self, dashboard_type: DashboardType, server: Flask) -> dash
 
     @staticmethod
     def _create_graphs(
-        report_by_short_name: Dict[str, CountryReport], prediction_event: PredictionEvent,
+        prediction_db: PredictionDb, report_by_short_name: Dict[str, CountryReport], prediction_event: PredictionEvent,
```

Comment:
Alternatively, this wouldn't be a static method anymore and would use `self.prediction_db`.

Reply:
You know I like it pure :)